src/js/containers/agencyV2/visualizations/TotalObligationsOverTimeContainer.jsx

```diff
@@ -68,7 +68,7 @@ const TotalObligationsOverTimeContainer = ({
     }, []);
 
     return (
-        <div ref={containerReference} className="total-obligations-over-time-visualization-container">
+        <div style={{ overflow: 'visible' }} ref={containerReference} className="total-obligations-over-time-visualization-container">
```

Comment:
Is it possible to apply this styling via CSS instead of inline?

Reply:
Done!